attestation-service/attestation-service/src/token/simple.rs

```diff
@@ -87,20 +107,58 @@ impl AttestationTokenBroker for SimpleAttestationTokenBroker {
     }
 
     fn pubkey_jwks(&self) -> Result<String> {
-        let pubkey = self.private_key.to_public_key();
-        let n = pubkey.n().to_bytes_be();
-        let e = pubkey.e().to_bytes_be();
+        let n = self.private_key.n().to_vec();
+        let e = self.private_key.e().to_vec();
+
+        let mut jwk = Jwk {
+            kty: "RSA".to_string(),
+            alg: SIMPLE_TOKEN_ALG.to_string(),
+            n: URL_SAFE_NO_PAD.encode(n),
+            e: URL_SAFE_NO_PAD.encode(e),
+            x5u: None,
+            x5c: vec![],
+        };
+
+        if let Some(signer) = &self.config.signer {
+            if let Some(cert_url) = signer.cert_url.clone() {
+                jwk.x5u = Some(cert_url);
+            }
+            if let Some(cert_path) = signer.cert_path.clone() {
```

Comment:
Maybe we can store the cert string inside the `SimpleAttestationTokenBroker` object to avoid read the filesystem everytime needs to export pubkey